registries.service

```diff
@@ -4,6 +4,7 @@ Before=docker.service
 After=network.target
 
 [Service]
+EnvironmentFile=/etc/sysconfig/docker
```

Comment:
I think you can add a -/etc/sysconfig/docker to this so that if the docker environment is not there, then it will still work.